tests/foreman/virtwho/test_api_virtwho.py

```diff
@@ -17,8 +17,12 @@
 from fauxfactory import gen_string
 from nailgun import entities
 from robottelo.config import settings
-from robottelo.decorators import tier2
+from robottelo.decorators import (
+    tier2,
+    skip_if_bug_open
```

Comment:
Bruno is working on removing this decorator: https://github.com/SatelliteQE/robottelo/issues/7336
I don't think you have to stop using it now (his PR is not yet merged), but something to be aware of.

Reply:
Thanks for the reminder, and I also want to remove it as our bug was ON_QA, we can test the case now